### PR TITLE
Git fixes

### PR DESCRIPTION
**Pull request by paulfioravanti**
I'm not sure how much of these changes would be welcome, but I figured I would try and submit some fixes anyway for issues that I found while doing the Git exercises on Typey Type.

If the Git dictionary is a bit more of a personal one, then I'm happy to have this PR closed.

### CODE REVIEW

File: dictionaries/git.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
 "TEUG": "tig",
 "TEUGS": "tig status",
 "TKPWEUT": "git",
-"AD/-P": "add .",
```
Comment (didoesdigital) [inline dictionaries/git.json:9]: Here, I'm building on the convention set up in the [code dictionary for `"-P": ".",`](https://github.com/didoesdigital/steno-dictionaries/blob/master/dictionaries/code.json#L67), and would prefer to keep that, as `PO*EUP` is the [code dictionary brief for `"{^}../{^}",`](https://github.com/didoesdigital/steno-dictionaries/blob/master/dictionaries/code.json#L94)

File: dictionaries/git.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
 "ST*TS": "status",
 "TKPW*EUTS": "{^}git status",
 "TKPW*TS": "{^}git status{#Return}",
-"HROG/H*PB/P*": "log -p",
```
Comment (didoesdigital) [inline dictionaries/git.json:14]: This builds on the convention for `"H*PB": "{-^}",` set up in the [code dictionary](https://github.com/didoesdigital/steno-dictionaries/blob/master/dictionaries/code.json#L73)

File: dictionaries/git.json [hunk 3]
```diff
@@ -19,7 +19,6 @@
 "TKPWEUT/KPHEUT/H-PBS/SR*": "git commit -v",
 "TKPWEUT/KPHEUT/H-PBS/SR*/TK-RB/APLD": "git commit -v --amend",
 "TKPWEUT/PHERPBLG": "git merge",
-"TKPWEUT/PURB/O*RPBLG/PHAFRT": "push origin master",
```
Comment (didoesdigital) [inline dictionaries/git.json:22]: 🙏 thank you

File: dictionaries/git.json [hunk 4]
```diff
@@ -17,13 +17,13 @@
 "TKPWEUT/KHEBG/O*UT": "git checkout",
 "TKPWEUT/KHEBG/O*UT/H-PBS/PW*": "git checkout -b",
 "TKPWEUT/KPHEUT/H-PBS/SR*": "git commit -v",
-"TKPWEUT/KPHEUT/H-PBS/SR*/TK-RB/APLD": "git commit -v --amend",
+"TKPWEUT/KPHEUT/H-PBS/SR*/H-PBZ/APLD": "git commit -v --amend",
 "TKPWEUT/PHERPBLG": "git merge",
 "TKPWEUT/ST*TS": "git status",
-"TKPWEUT/TKEUF/TK-RB/KAERBD": "git diff --cached",
```
Comment (didoesdigital) [inline dictionaries/git.json:24]: These two could be changed to `TK*RB` using the code dictionary, or `H-PBZ` as you've suggested, which matches Plover. Happy to run with that.

File: dictionaries/git.json [hunk 5]
```diff
@@ -35,7 +35,7 @@
 "TKPWH*TD": "{^}git commit -v --amend{#Return}{^}i{^}{-|}",
 "TKPWEUT/PURB/O*RPBLG/PHAFRT": "git push origin master",
 "TKPWEUT/PURB/TPORS": "git push --force-with-lease",
-"PUL/TK*RB RE/PWAEUS": "pull --rebase",
```
Comment (didoesdigital) [inline dictionaries/git.json:39]: Good catch 👍

File: dictionaries/git.json (hunk 1 above)
Comment (paulfioravanti) [reply dictionaries/git.json:9]: Okay, understood. I'll remove the commit that adds this.

File: dictionaries/git.json (hunk 2 above)
Comment (paulfioravanti) [reply dictionaries/git.json:14]: Okay, I'll remove the commit that changes single hyphen flag usage from `H*PB` to `H-PBS`.

File: dictionaries/git.json (hunk 4 above)
Comment (paulfioravanti) [reply dictionaries/git.json:24]: Sounds good. So, I'll leave all the entries where `--` has been changed to `H-PBZ` for standardisation purposes.

**Pull request merged**